src/languageServerInstaller.ts

```diff
@@ -19,10 +24,30 @@ export class LanguageServerInstaller {
 	private userAgent = `Terraform-VSCode/${extensionVersion} VSCode/${vscode.version}`;
 	private release: Release;
 
-	public async needsInstall(): Promise<boolean> {
+	private async getRequiredVersionRelease(versionString: string): Promise<Release> {
 		try {
-			this.release = await getRelease("terraform-ls", "latest", this.userAgent);
+			const release = await getRelease("terraform-ls", versionString, this.userAgent);
+			console.log(`Found Terraform language server version ${release.version} which satisfies range '${versionString}'`);
+			return release;
 		} catch (err) {
+			if (versionString == defaultVersionString) { throw err; }
+			console.log(`Error while finding Terraform language server release which satisfies range '${versionString}' and will reattempt with '${defaultVersionString}': ${err}`)
```

Comment:
```suggestion
			console.log(`Error while finding Terraform language server release which satisfies range '${versionString}' and will reattempt with '${defaultVersionString}': ${err}`)
			vscode.window.showWarningMessage(`No version matching ${versionString} found, searching for ${defaultVersionString} instead`);
```

I think this would be a valuable feedback to the end user also.